server/main.py

```diff
@@ -12,11 +15,32 @@
     },
 )
 
+def printok():
+    print("Everything is fine")
+
+def liveness():
+    try:
+        printok()
+    except Exception:
+        raise HealthError("Can't connect to the file")
+
+def readiness():
+    try:
+        printok()
+    except Exception:
+        raise HealthError("Can't connect to the file")
+
+app.config.update(
+    HEALTHZ = {
+        "alive": "main.liveness",
+        "ready": "main.readiness",
+    }
```

Comment:
Could you briefly explain what these two things stand for in our application?

Reply:
I am just using [`flask-healthz` package](https://pypi.org/project/flask-healthz/) here. 

It simply maps two routes `/alive`, `/ready` to `liveness` and `readiness` functions. If you look inside each function, there is `try ... catch` phrase. If any exception never happens, it will return `OK (200)` status to the client. Otherwise some error code. 

It can be simply implemented by declaring the routes by ourselves, but I chose to use this package since it is more comprehensible I guess (like `try ... catch`).

Reply:
I see. Thanks.